### PR TITLE
t/10: Allowed the editable element in DecoupledEditor.constructor(). Removed config.toolbarContainer and config.editableContainer.

### LINKED ISSUE #10: A bit confusing editor initialization.
This is probably due to lack of docs or just that other

The confusing part might be with how to set data for editing in the `DecoupledEditor.create()` method as it kinda works differently then with other editors.

Th consfusing part was that `editableContainer` cannot be used to init data to edit (despite the fact that there's a param for feeding the editor with data).

Also the confusing part is that if there's anything in the `editableContainer` it will be around editable area but will not be editable.

selection_030

Now the confusion probably came from that other editors used some container as both editable container and to feed data to the editor.

### PR DESCRIPTION
**Pull request by oleq**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Other: Allowed the editable element to be passed into `DecoupledEditor.constructor()`. Removed `config.toolbarContainer` and `config.editableContainer`. Closes #10. Closes ckeditor/ckeditor5#912.

BREAKING CHANGE: The config options `config.toolbarContainer` and `config.editableContainer` have been removed. Please refer to the `DecoupledEditor` class API documentation to learn about possible methods of bootstrapping the UI.

---

### Additional information

It's a constellation:
* https://github.com/ckeditor/ckeditor5-build-decoupled-document/pull/5
* https://github.com/ckeditor/ckeditor5/pull/925 (docs, mgit.json)

### CODE REVIEW

File: docs/framework/guides/document-editor.md [hunk 1]
```diff
@@ -58,9 +61,13 @@ The code you have just created will run the editor but still, the user interface
 The following structure has two containers that correspond to the configuration you have just used. The editor will inject the toolbar and editable into respective containers as it starts.
 
 ```html
-<div class="document-editor">
+<div class="document-editor ck-rounded-corners">
```
Comment (Reinmar) [inline docs/framework/guides/document-editor.md]: Why is it required to add this class manually? This is unclear for me why, as an integration developer, I'd need to add some CKE's class to my elements to make buttons in the toolbar rounded.

**Comment by Reinmar** [comment]
It's very odd, but the builds are constantly failing: see e.g. https://travis-ci.org/ckeditor/ckeditor5-editor-decoupled/builds/359460250

Chrome runs ok, but then the 3 other browsers (local FF and BrowserStack's Edge and Safari) crash:

image

They crash somewhere around 13th test. Crash or timeout, because the timing is weird too.

When running these tests locally once or twice I got them to crash too. But this is oooood.

**Comment by Reinmar** [comment]
Unfortunately, R- due to failing tests.

File: docs/framework/guides/document-editor.md (hunk 1 above)
Comment (oleq) [reply docs/framework/guides/document-editor.md]: Fixed in https://github.com/ckeditor/ckeditor5-editor-decoupled/pull/11/commits/63683b6e1ba041607fb830b0c7896a5a0c88a07c.

**Pull request merged**